lib/libfigpar/string_m.c

```diff
@@ -24,11 +24,8 @@
  * SUCH DAMAGE.
  */
 
-#include <sys/types.h>
```

Comment:
This is actually needed. But stdint.h is the new way to spell this for uintXX_t that this file uses. I'll fix.